### ISSUE TITLE
sdkman

### ISSUE THREAD

**Issue by xenoterracide** [romkatv/powerlevel10k#1536]
as an alternative to jenv, it would be nice to support [sdkman](https://sdkman.io/) the big difference being this

from the jenv readme

> However, this project does not:
>    Install java for you. Use your platform appropriate package manager to install java. On macOS, brew is recommended.

sdkman does in fact install java for you. Technically the java version will actually resolve

**Comment by romkatv** [comment]
If you send a high quality PR, I'll merge it. Otherwise this may never happen.

**Comment by xenoterracide** [comment]
It may never happen. To be honest, I was going through the p10k file after this, and saw `asdf`  and was like “what's that?” after looking it up, I've switched to that, although the java plugin needs a bit of improvement. I decided ultimately though that I'd rather use the `node_version` and `java_version` for display anyway, s `lts` for node is not enough information, and `adoptopenjdk......` is too much. We'll see if I stick with it over nvm/sdkman. Certainly though it's a better idea than having 200 different language managers.
